ent/src/yb/tools/yb-admin_client_ent.cc

```diff
@@ -143,14 +143,21 @@ Status ClusterAdminClient::ListSnapshots(bool show_details) {
   ListSnapshotRestorationsResponsePB rest_resp;
   RETURN_NOT_OK(master_backup_proxy_->ListSnapshotRestorations(rest_req, &rest_resp, &rpc));
 
+  if (not_show_restored) {
+    cout<< "Not show fully RESTORED entries." << endl;
+  }
+
   if (rest_resp.restorations_size()) {
     cout << RightPadToUuidWidth("Restoration UUID") << kColumnSep << "State" << endl;
   } else {
     cout << "No snapshot restorations" << endl;
   }
 
   for (const SnapshotInfoPB& snapshot : rest_resp.restorations()) {
-    cout << SnapshotIdToString(snapshot.id()) << kColumnSep << snapshot.entry().state() << endl;
+    if (!not_show_restored ||
+        snapshot.entry().state() != master::SysSnapshotEntryPB_State_RESTORED) {
```

Comment:
Use here `SysSnapshotEntryPB::RESTORED`
plus `using master::SysSnapshotEntryPB;` in the top of the module.